api/src/main/resources/workbench.yaml

```diff
@@ -1509,14 +1524,12 @@ definitions:
     type: string
     description: actions a user can have authority/permission to perform
     enum: [
+        # Note: Swagger trims any common prefix from enum values' corresponding
+        # Java fields by default; this has no side-effect currently as there is
+        # no common prefix. https://github.com/swagger-api/swagger-codegen/issues/4261
+        AUDIT,
```

Comment:
Rather than using authorities for this stuff, we may want to consider this sort of thing:

https://cloud.google.com/appengine/docs/standard/java/config/cron#securing_urls_for_cron

We don't really intend for these endpoints to be invoked by end users... right?

Reply:
Yep, true. I was imagining there would be a manual use case, but I don't see why that couldn't just be someone clicking the "launch now" button in cloud console. Changed to check the header. I couldn't figure out how to generally access HTTP headers in this context, so I wound up just adding this to the swagger which is probably for the best anyways.

I'm not sure if this request will actually be authenticated or not, so I've dropped that requirement (let me know if that sounds right). And also moved this under /cron since it has a different usage pattern / constraints.